components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -108,3 +109,54 @@ assertEquals(6, Controls.refreshPressed.testGetValue())
 
 NOTE: Using the testing API requires calling `Glean.enableTestingMode()` first,
 such as from a `@Before` method.
+
+## Timing Distributions
+
+Used to accumulate and store time measurement samples into buckets, which are exponentially sized 
+using a natural log function. 
+
+If you wanted to create a timing distribution to measure page load times, first you need to add 
+an entry for the counter to the `metrics.yaml` file:
+
+```YAML
+pages:
+  page_load:
+    type: timing_distribution
+    time_unit: millisecond
+    description: >
+      Counts how long each page takes to load into exponential buckets
+    ...
+```
+
+Note that `time_unit` is required in the `metrics.yaml` file to identify the base unit of time
+used to indicate the size of the buckets and the units intended when accumulating and recording
+samples.
```

Comment:
Can we avoid mentioning buckets?
E.g. *"`time_unit` is required and allows for values like `millisecond` and ... Which range of values is recorded in detail depends on the `time_unit`, e.g. for `milliseconds` all values greater `60000` are recorded as overflow values.*
(Not sure i got the values right)